www-apps/prowlarr/files/prowlarr.logrotate

```diff
@@ -1,4 +1,4 @@
-/var/lib/prowlarr/.config/prowlarr/logs/*.log{
+/var/lib/prowlarr/.config/Prowlarr/logs/*.txt{
```

Comment:
Please revbump for these fixes (git mv is fine, as it's all binary anyway).

Reply:
@thesamesam you mean revbumping ebuilds themselves and not logrotate files, right?

Reply:
yes, sorry :)